x-pack/plugins/uptime/server/rest_api/monitors/monitors_durations.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { UMServerLibs } from '../../lib/lib';
+import { UMRestApiRouteFactory } from '../types';
+
+export const createGetMonitorDurationRoute: UMRestApiRouteFactory = (libs: UMServerLibs) => ({
+  method: 'GET',
+  path: '/api/uptime/monitor/duration',
+  validate: {
+    query: schema.object({
+      monitorId: schema.string(),
+      dateStart: schema.maybe(schema.string()),
+      dateEnd: schema.maybe(schema.string()),
```

Comment:
```suggestion
      dateStart: schema.string(),
      dateEnd: schema.string(),
```

Should the date range be optional? It looks like it's required by the request function we call to fetch the data.

Reply:
done